src/umpire/util/FixedMallocPool.cpp

```diff
@@ -15,6 +15,7 @@
 #include <cstring>
 #include <cstdlib>
 #include <algorithm>
+#include <cstddef>
```

Comment:
It appears that this was the only change to this file.  Was there a side-effect change in a different file that caused this one to need to change?

Reply:
Needs `std::size_t` - removing the `Allocator.hpp` file below meant this is now required.